packages/runner/src/app/app.jsx

```diff
@@ -22,6 +22,7 @@ class App extends Component {
 
   render () {
     const specPath = this.props.util.absoluteSpecPath(this.props.config)
+    const relativeSpecPath = decodeURI(this.props.util.relativeSpecPath(this.props.config))
```

Comment:
I'm still seeing the issue with special characters. Can you add an integration test covering this? Have it include `?`, `&`, and `%`, since those are the ones [we have special handling for](https://github.com/cypress-io/cypress/blob/develop/packages/server/lib/util/escape_filename.ts). I think the solution is to use `decodeURIComponent` instead of `decodeURI`.

Reply:
@chrisbreiding  I've made this change but I am unsure of how to proceed regarding tests. Within the existing integration tests, the reporter is rendered directly without being launched from the runner. It seems like this would require an `e2e` test, but I'm pretty unfamiliar with how those work.